pkg/systemd/services/timer-dialog.jsx

```diff
@@ -356,7 +356,7 @@ const CreateTimerDialogBody = ({ owner }) => {
                                                         buttonAriaLabel={_("Toggle date picker")}
                                                         locale={timeformat.dateFormatLang()}
                                                         weekStart={timeformat.firstDayOfWeek()}
-                                                        onChange={(str, data) => {
+                                                        onChange={(_, str, data) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18222-20230123-142319-6860d3f9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)